### PR TITLE
OnlyTitle and OnlySearch Modes added

### PR DESCRIPTION
**Pull request by yamanogluberk**
Search Icon added,
OnlyTitle mode added for home pages with only title on top like profile page
OnlySearch mode added for pages that might have a search option

### CODE REVIEW

File: application/android/app/src/main/java/com/group12/beabee/views/MainStructure/BaseContainerFragment.java [hunk 1]
```diff
@@ -143,6 +152,27 @@ private void ModeEditable() {
...
+        topbar.setVisibility(View.VISIBLE);
+        btnBack.setVisibility(View.GONE);
+        btnCancel.setVisibility(View.GONE);
+        btnApprove.setVisibility(View.GONE);
+        btnEdit.setVisibility(View.GONE);
+        btnAdd.setVisibility(View.GONE);
+        btnSearch.setVisibility(View.GONE);
+    }
+
+    private void ModeOnlySearch() {
```
Comment (Refref18) [inline application/android/app/src/main/java/com/group12/beabee/views/MainStructure/BaseContainerFragment.java:168]: Only Search can be used both in user @ihsangkcl and market place search, thanks.

**Review by Refref18 — CHANGES_REQUESTED** [review]

**Review by ihsangkcl — APPROVED** [review]
It looks fine to me. I can use OnlySearch mode for user search page. Thanks

File: application/android/app/src/main/java/com/group12/beabee/views/MainStructure/BaseContainerFragment.java [hunk 2]
```diff
@@ -105,6 +107,12 @@ public void SetMode(PageMode pageMode) {
             case OnlyBack:
                 ModeOnlyBack();
                 break;
+            case OnlyTitle:
+                ModeOnlyTitle();
+                break;
+            case OnlySearch:
+                ModeOnlySearch();
+                break;
```
[the remark this answers is not in the document]
Comment (yamanogluberk) [reply application/android/app/src/main/java/com/group12/beabee/views/MainStructure/BaseContainerFragment.java:119]: It is added with new commit

[the remark this answers is not in the document]
Comment (Refref18) [reply application/android/app/src/main/java/com/group12/beabee/views/MainStructure/BaseContainerFragment.java:119]: Thank you, it looks completed with the new commit.

**Review by Refref18 — APPROVED** [review]

**Pull request merged**